src/SprykerEco/Zed/AvalaraTax/Persistence/Propel/AbstractSpyTaxAvalaraApiLog.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
```

Comment:
* Switch to the MIT license header.

/**
 * MIT License
 * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
 */

Reply:
Resolved.